sparse/client.go

```diff
@@ -285,6 +293,7 @@ func (client *syncClient) syncDataInterval(file FileIoProcessor, dataInterval In
 			}
 		}
 		offset += batchInterval.Len()
+		ops.UpdateSyncFileProgress(size)
```

Comment:
Size? Are we going to use the accumulated result?

Reply:
Yes.